conjur/api/client.py

```diff
@@ -17,13 +17,14 @@
 from conjur.api.api import Api
 from conjur.config import Config as ApiConfig
 from conjur.constants import DEFAULT_NETRC_FILE
-from conjur.controllers.init_controller import InitController
-from conjur.logics.init_logic import InitLogic
-from conjur.data_objects.conjurrc_data import ConjurrcData
+from conjur.controller.init_controller import InitController
+from conjur.logic.init_logic import InitLogic
+from conjur.data_object.conjurrc_data import ConjurrcData
 from conjur.wrappers.netrc_wrapper import NetrcWrapper
```

Comment:
why netrc_wrapper? 
I think we should rethink this name two fold 1. should we call it netrc? something so so specific 2. should it be a wrapper?

Please explain what you think here (:

Reply:
Renamed to CredentialsFromFile  and moved to logic folder